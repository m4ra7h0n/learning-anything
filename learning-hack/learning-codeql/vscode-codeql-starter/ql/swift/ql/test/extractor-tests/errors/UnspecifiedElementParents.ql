import swift

from UnspecifiedElement e, Locatable parent
where parent = e.getParent()
select parent, e
