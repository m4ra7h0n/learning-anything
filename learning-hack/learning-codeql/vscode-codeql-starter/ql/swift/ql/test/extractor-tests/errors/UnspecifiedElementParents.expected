| unspecified.swift:3:1:3:23 | extension | unspecified.swift:3:1:3:23 | missing extended_type_decl from ExtensionDecl |
| unspecified.swift:9:9:9:9 | fallthrough | unspecified.swift:9:9:9:9 | missing fallthrough_dest from FallthroughStmt |
| unspecified.swift:9:9:9:9 | fallthrough | unspecified.swift:9:9:9:9 | missing fallthrough_source from FallthroughStmt |
| unspecified.swift:12:18:12:21 | (no string representation) | unspecified.swift:12:18:12:21 | missing element from EnumElementPattern |
| unspecified.swift:14:18:14:26 | (no string representation) | unspecified.swift:14:18:14:26 | missing element from EnumElementPattern |
| unspecified.swift:19:18:19:19 | (no string representation) | unspecified.swift:19:18:19:19 | missing element from EnumElementPattern |
| unspecified.swift:22:13:22:13 | fallthrough | unspecified.swift:22:13:22:13 | missing fallthrough_dest from FallthroughStmt |
| unspecified.swift:22:13:22:13 | fallthrough | unspecified.swift:22:13:22:13 | missing fallthrough_source from FallthroughStmt |
| unspecified.swift:26:18:26:19 | (no string representation) | unspecified.swift:26:18:26:19 | missing element from EnumElementPattern |
